rpc-protocol/src/main/java/org/sonarsource/sonarlint/core/rpc/protocol/backend/connection/org/FuzzySearchUserOrganizationsParams.java

```diff
@@ -30,10 +30,12 @@ public class FuzzySearchUserOrganizationsParams {
   @JsonAdapter(EitherCredentialsAdapterFactory.class)
   private final Either<TokenDto, UsernamePasswordDto> credentials;
   private final String searchText;
+  private final String region;
 
-  public FuzzySearchUserOrganizationsParams(Either<TokenDto, UsernamePasswordDto> credentials, String searchText) {
+  public FuzzySearchUserOrganizationsParams(Either<TokenDto, UsernamePasswordDto> credentials, String searchText, String region) {
```

Comment:
This is an API, so you should also introduce a constructor overload and deprecate the previous one